### DISCUSSION TITLE
Is CREATE EVENT supported?

### DISCUSSION THREAD

**Discussion by isaac-mcfadyen** [planetscale/discussion#225] [Q&A] [answered]
When doing a `CREATE EVENT` using Planetscale, a syntax error is thrown near `EVENT`.

However, doing a `SHOW processlist` shows the event scheduler running and active.

Is MySQL's Event Scheduler supported on Planetscale?

**Comment by jgreet** [comment, marked as answer]
Sorry, stored routines are not supported on PlanetScale databases.

You can check out our MySQL Compatibility documentation here:
https://docs.planetscale.com/reference/mysql-compatibility
